OpenTelemetryApi.podspec

```diff
@@ -0,0 +1,24 @@
+Pod::Spec.new do |spec|
+  spec.name = "OpenTelemetryApi"
+  spec.version = "1.10.1"
```

Comment:
We'll need to make sure these versions get bumped when performing releases. Would hope that could be automated in the existing release process.